packages/frontend/src/components/SchedulersView/LogsView.tsx

```diff
@@ -0,0 +1,339 @@
+                    );
+                    const chartOrDashboard = item.savedChartUuid
+                        ? charts.find(
+                              (chart) =>
+                                  chart.savedChartUuid === item.savedChartUuid,
+                          )
+                        : dashboards.find(
+                              (dashboard) =>
+                                  dashboard.dashboardUuid ===
+                                  item.dashboardUuid,
+                          );
+                    return (
+                        <Anchor
+                            sx={{
+                                color: 'unset',
+                                ':hover': {
+                                    color: 'unset',
+                                    textDecoration: 'none',
+                                },
+                            }}
+                            href={getSchedulerLink(item, projectUuid)}
+                            target="_blank"
+                        >
+                            <Group noWrap>
+                                {getSchedulerIcon(item, theme)}
+                                <Stack spacing="two">
+                                    <Tooltip
+                                        label={
+                                            <Stack spacing="two">
+                                                <Text fz={12} color="gray.5">
+                                                    Schedule type:{' '}
+                                                    <Text color="white" span>
+                                                        {item.format === 'csv'
+                                                            ? 'CSV'
+                                                            : 'Image'}
+                                                    </Text>
+                                                </Text>
+                                                <Text fz={12} color="gray.5">
+                                                    Created by:{' '}
+                                                    <Text color="white" span>
+                                                        {user?.firstName}{' '}
+                                                        {user?.lastName}
+                                                    </Text>
+                                                </Text>
+                                            </Stack>
+                                        }
+                                    >
+                                        <Text
+                                            fw={600}
+                                            lineClamp={1}
+                                            sx={{
+                                                overflowWrap: 'anywhere',
+                                                '&:hover': {
+                                                    textDecoration: 'underline',
```

Comment:
it feels like you are trying to add an anchor to the whole cell and only apply the hover link UX to the name. should we consider making this component an anchor and removing it from the whole cell?